libs/platform/src/lib/components/approval-flow/approval-flow-node/approval-flow-node.component.ts

```diff
@@ -47,11 +56,29 @@ export class ApprovalFlowNodeComponent implements OnInit, OnChanges {
     /** Whether node element has carousel end marker. Should be set to 'true' for the last node */
     @Input() renderCarouselEndMarker = false;
 
+    /** Whether to display add node button before in Edit mode */
+    @Input() canAddNodeBefore = false;
+
+    /** Whether to display add node button after in Edit mode */
+    @Input() canAddNodeAfter = false;
```

Comment:
Is there a difference between these properties, and the properties of the same name in `ApprovalGraphNodeMetadata` ? Is there a reason to have these `@Input`s and `@Input() meta`?

Reply:
thanks, I removed these properties, using `meta` is enough